test/Annotation/AnnotationBuilderTest.php

```diff
@@ -242,7 +241,7 @@ public function testAllowsComposingMultipleChildEntities()
      * @dataProvider provideOptionsAnnotationAndComposedObjectAnnotation
      * @param string $childName
      *
-     * @group 7108
+     * @group laminas7108
```

Comment:
These changes are unnecessary because the (most) numbers are from the [old Zend Framework repository](https://github.com/zendframework/zendframework/pull/7108) and we have no naming scheme for groups.

Thanks in advance!

Reply:
I made these changes because PHPUnit errored saying bare numbers are not accepted as group name.
But it seems I was wrong, since it's not erroring anymore now :|

Reply:
I was right: reverting all `@group laminas(\d+)` I get:

```
$ vendor/bin/phpunit 
PHP Fatal error:  Uncaught TypeError: strpos() expects parameter 1 to be string, int given in ./laminas-form/vendor/phpunit/phpunit/src/Framework/TestSuite.php:899
Stack trace:
#0 ./laminas-form/vendor/phpunit/phpunit/src/Framework/TestSuite.php(899): strpos()
#1 ./laminas-form/vendor/phpunit/phpunit/src/Framework/TestSuite.php(287): PHPUnit\Framework\TestSuite->containsOnlyVirtualGroups()
#2 ./laminas-form/vendor/phpunit/phpunit/src/Framework/TestSuite.php(365): PHPUnit\Framework\TestSuite->addTest()
#3 ./laminas-form/vendor/phpunit/phpunit/src/Framework/TestSuite.php(504): PHPUnit\Framework\TestSuite->addTestSuite()
#4 ./laminas-form/vendor/phpunit/phpunit/src/Framework/TestSuite.php(529): PHPUnit\Framework\TestSuite->addTestFile()
#5 ./laminas-form/vendor/phpunit/phpunit/src/TextUI/XmlConfiguration/TestSuite/TestSuiteMapper.php(50): PHPUnit\Framework\T in ./laminas-form/vendor/phpunit/phpunit/src/TextUI/Command.php on line 103
```

Reply:
This does not change the fact that `laminas` is wrong for this groups. 😉 
I would use something general like `issue`.

Reply:
Done